horizon/enforcer/api.py

```diff
@@ -329,16 +307,13 @@ async def authorized_users(
     async def is_allowed_url(
         request: Request,
         query: UrlAuthorizationQuery,
-        x_permit_sdk_language: Optional[str] = Depends(notify_seen_sdk),
+        x_permit_sdk_language: Annotated[str | None, Depends(notify_seen_sdk)],
     ):
         data = await post_to_opa(request, "mapping_rules", None)
 
         mapping_rules = []
         data_result = json.loads(data.body).get("result")
-        if data_result is None:
-            mapping_rules_json = []
-        else:
-            mapping_rules_json = data_result.get("all")
+        mapping_rules_json = data_result.get("all", []) if data_result is not None else []
```

Comment:
if data_result is 
```json
{
    "all": null
}
```
then the mapping_rules_json will be None

Reply:
There was a discussion about it back then, I can't remember what the understanding was. 
https://github.com/permitio/PDP/pull/226#discussion_r1890525659

What would you suggest doing to make it clearer?

Reply:
Fixed